osu.Game/Screens/Play/PlayerSettings/PlaybackSettings.cs

```diff
@@ -24,40 +30,127 @@ public partial class PlaybackSettings : PlayerSettingsGroup
+                                {
+                                    Anchor = Anchor.Centre,
+                                    Origin = Anchor.Centre,
+                                    Icon = FontAwesome.Solid.Forward,
+                                    Action = () => seek(1, seek_amount),
+                                },
+                                new IconButton
+                                {
+                                    Anchor = Anchor.Centre,
+                                    Origin = Anchor.Centre,
+                                    Icon = FontAwesome.Solid.FastForward,
+                                    Action = () => seek(1, seek_fast_amount),
+                                },
+                            },
                         },
-                        multiplierText = new OsuSpriteText
+                        new Container
                         {
-                            Anchor = Anchor.CentreRight,
-                            Origin = Anchor.CentreRight,
-                            Font = OsuFont.GetFont(weight: FontWeight.Bold),
-                        }
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Children = new Drawable[]
+                            {
+                                rateSlider = new PlayerSliderBar<double>
+                                {
+                                    LabelText = "Playback speed",
+                                    Current = UserPlaybackRate,
+                                },
+                                multiplierText = new OsuSpriteText
+                                {
+                                    Anchor = Anchor.TopRight,
+                                    Origin = Anchor.TopRight,
+                                    Font = OsuFont.GetFont(weight: FontWeight.Bold),
+                                    Margin = new MarginPadding { Right = 20 },
+                                }
+                            },
+                        },
                     },
                 },
-                rateSlider = new PlayerSliderBar<double> { Current = UserPlaybackRate }
             };
+
+            void seek(int direction, double amount)
+            {
+                double target = Math.Clamp((gameplayClock?.CurrentTime ?? 0) + (direction * amount), 0, gameplayState?.Beatmap.GetLastObjectTime() ?? 0);
+                gameplayClock?.Seek(target);
+            }
         }
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
             rateSlider.Current.BindValueChanged(multiplier => multiplierText.Text = $"{multiplier.NewValue:0.0}x", true);
         }
+
+        protected override void Update()
+        {
+            base.Update();
+            play.Icon = gameplayClock?.IsRunning == true ? FontAwesome.Regular.PauseCircle : FontAwesome.Regular.PlayCircle;
```

Comment:
i'm 99.100% sure there's a `IsPaused` bindable which will give a much better reading (ie. make the icon update immediately).